### ISSUE TITLE
tflite_runtime interpreter produces strange results when warmup is done with empty images

### ISSUE THREAD

**Issue by FSet89** [tensorflow/tensorflow#49370]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): YOCTO Linux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.4.1
- Python version: 3.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the current behavior**
I have an application that runs on an embedded system. The application acquires images from a camera and performs predictions using a custom quantized neural network and the tflite_runtime interpreter. The applications and the model works fine with good accuracy, but the first inferences are very slow because of the warmup time needed by the interpreter. For this reason I tried to perform the warmup before the application starts, by feeding the interpreter some empty images (numpy.zeros). However, this causes subsequent inferences with real images to produce almost constant (and wrong) predictions.

**Describe the expected behavior**
I expect the interpreter to perform well even if the warmup is done with empty images

**[Contributing](https://www.tensorflow.org/community/contribute)** - Do you
want to contribute a PR? (yes/no): no - Briefly describe your candidate solution
(if contributing): 

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

```
import tflite_runtime.interpreter as tflite
import numpy as np
import cv2

def classify(interpreter, input_details, output_details, img):
	input_shape = input_details[0]['shape']
	img = cv2.resize(img, (input_shape[2], input_shape[1]))
	img = (img/255.0).astype(np.float32)
	img = np.expand_dims(img, 0)
	interpreter.set_tensor(input_details[0]['index'], img)	
	interpreter.invoke()			
	output = interpreter.get_tensor(output_details[0]['index'])
	output = np.squeeze(output)
        return output


interpreter = tflite.Interpreter('mymodel.tflite')
interpreter.allocate_tensors()
input_details = interpreter.get_input_details()
output_details = interpreter.get_output_details()
input_shape = input_details[0]['shape']

warmup_img = np.zeros(input_shape)
_ = classify(interpreter, input_details, output_details, warmup_img)

cap = cv2.VideoCapture(0)

while cap.isOpened():
      _, frame = cap.read()
      print(classify(interpreter, input_details, output_details, frame))
```

	

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by abattery** [comment]
Could you provide the TFLite model or a minimal and reproducible step as a gist?
